### DISCUSSION TITLE
Removing noise in doctr

### DISCUSSION THREAD

**Discussion by navaneeth-nagu** [mindee/doctr#1656] [Q&A] [answered]
Hi team, 

Does doctr remove noise in the document or if there is a better way to remove the noise? Please suggest.

Thanks in advance.

**Comment by felixdittrich92** [comment, marked as answer]
Hi @navaneeth-nagu :wave:,

The only things we do by default are resizing the inputs and normalizing so no we don't remove noise :)

```
multi_img_doc = DocumentFile.from_images(["path/to/page1.jpg", "path/to/page2.jpg"])
```
The DocumentFile returns a list of numpy arrays so you could do with it whatever you want (for example any OpenCV operation) before passing the list of np arrays to the model/pipeline

Or you use a AutoEncoder to remove noise i have worked on something a few years ago (maybe useful as reference):
https://github.com/felixdittrich92/Document_Scanner#denoising-image

**Reply by navaneeth-nagu** [reply]
Hi @felixdittrich92 👋,

Thanks. This will definitely help. :)
